metpy/io/upperair.py

```diff
@@ -155,6 +155,12 @@ def get_data(time, site_id, region='naconf'):
                              'for station {stid}.'.format(time=time, region=region,
                                                           stid=site_id))
 
+        # See if the return is valid, but has no data
+        if data == b'Sorry, the server is too busy to process your request.\n' \
```

Comment:
Any reason not to just do `if b'too busy' in data`?

Another option would be to just handle this after we look for the PRE tags.